sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/WriteToDataSourceV2Exec.scala

```diff
@@ -288,6 +324,12 @@ case class WriteToDataSourceV2Exec(
     copy(query = newChild)
 }
 
+// TODO: could we parameterize this? Not that easy.
```

Comment:
Any ideas will be appreciated. I'll spend more time thinking about this but didn't want to block the PR.